test/factories/host_related.rb

```diff
@@ -48,7 +61,7 @@ def set_nic_attributes(host, attributes, evaluator)
   factory :nic_base, :class => Nic::Base do
     type 'Nic::Base'
     sequence(:identifier) { |n| "eth#{n}" }
-    sequence(:mac) { |n| "00:00:00:00:" + n.to_s(16).rjust(4, '0').insert(2, ':') }
```

Comment:
Yeah, I'm less than clear why they were required there, which I suppose is what I was getting at!